--- cinnamon-screensaver-2.6.3/configure.ac.orig	2015-06-04 07:46:11.089336633 +0700
+++ cinnamon-screensaver-2.6.3/configure.ac	2015-06-04 07:46:41.493930910 +0700
@@ -58,7 +58,6 @@
         gtk+-3.0 >= $GTK_REQUIRED_VERSION
         dbus-glib-1 >= $DBUS_REQUIRED_VERSION
         cinnamon-desktop >= $CINNAMON_DESKTOP_REQUIRED_VERSION)
-PKG_CHECK_MODULES([WEBKIT], [webkit2gtk-3.0],, [PKG_CHECK_MODULES([WEBKIT], [webkit2gtk-4.0])])
 AC_SUBST(CINNAMON_SCREENSAVER_CFLAGS)
 AC_SUBST(CINNAMON_SCREENSAVER_LIBS)
 
